test/jdk/jdk/jfr/threading/TestDeepVirtualStackTrace.java

```diff
@@ -42,8 +42,9 @@
  * @requires vm.hasJFR & vm.continuations
  * @library /test/lib /test/jdk
  * @modules jdk.jfr/jdk.jfr.internal
- * @compile --enable-preview -source ${jdk.version} TestDeepVirtualStackTrace.java
- * @run main/othervm --enable-preview -XX:FlightRecorderOptions:stackdepth=2048
+ * @enablePreview
+ * @compile TestDeepVirtualStackTrace.java
```

Comment:
Is compilation needed here?

Reply:
For the two tests that compiled a single file, I remove the @compile directive. Any additional refactoring left for future work.